data/base-spells.yaml

```diff
@@ -1596,6 +1601,16 @@ spell_data:
     abbrev: PD
     mana: 2
     mana_type: lunar
+  Pattern Hues:
+    abbrev: p h
+    prep_type: prep cantrip
+    prep_time: 0
+    cast: gesture phosphorescent emerald green
+    mana:
```

Comment:
Does this being nil give any type of issue?

Reply:
No issue with mana nil.